modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -6,51 +6,69 @@ endif::[]
 
-image::munit-coverage-report-in-anypoint-studio.png[]
+image::munit-coverage-report-in-anypoint-studio.png["Files included in the report are highlighted."]
 
 [%header,cols="20a,80a"]
 |===
 | Column      | Description
-| *Resource*    | The name of the Mule Configuration file.
-| *Containers* | Number of Flows inside the Mule configuration file.
-| *Weight*      | This field represents how much of your application is contained inside a configuration file. +
+| *Resource*    | The name of the Mule Configuration file
+| *Containers* | Number of flows inside the Mule configuration file
+| *Weight*      | The percentage of your application in a configuration file. +
                 It is calculated as a percentage of the overall number of event processors in the documentation and the
-                amount of event processors contained in the Mule configuration file.
-| *Coverage*    | This field represents the percentage of the event processors inside the Mule Configuration file that
-                were executed in the MUnit test.
+                amount of event processors in the Mule configuration file.
+| *Coverage*    | The percentage of the event processors inside the Mule Configuration file that is executed in the MUnit test
 |===
 
-Additionally you can click in each Resource, to get a more specific and granular report of the Event Processors executed in that resource:
+Additionally, you can click on each *Resource* to get a more specific and granular report of the Event Processors executed in that resource:
 
-image::munit-granular-coverage-report-in-anypoint-studio.png[]
+image::munit-granular-coverage-report-in-anypoint-studio.png["Granular report is highlighted."]
 
 [%header,cols="20a,80a"]
 |===
 | Column              | Description
-| *Name*                | The name of the container.
-| *Type*                | The type of the container.
-| *Covered Processors* | The processors inside the container that were executed during the MUnit test.
-| *Processors*         | The total amount of processors inside the container.
-| *Coverage*            | This field represents the percentage of the event processors inside the container that were executed in the MUnit test.
+| *Name*                | The name of the container
+| *Type*                | The type of the container
+| *Covered Processors* | The processors inside the container that is executed during the MUnit test
+| *Processors*         | The total amount of processors inside the container
+| *Coverage*            | The percentage of the event processors inside the container executed in the MUnit test
 |===
 
 == Disable Coverage Reports
 
-If you choose so, you can disable the coverage calculation from your project's run configuration.
+If you choose so, you can disable the coverage calculation from your project run configuration.
```

Comment:
Why would you disable it? Give an example.

Reply:
It may be the case that developers want to test their flow but they do not want to have a coverage report, so they can disable this feature. This feature is not mandatory for testing a flow.

Reply:
Again, good information to share with the user.

Reply:
OK! Thanks!